### PR TITLE
Feature external feed dev anushika

### PR DESCRIPTION
**Pull request by anushika99**

### CODE REVIEW

File: projects/commudle-admin/src/app/feature-modules/external-feed/components/feed-item-details/feed-item-details.component.ts [hunk 1]
```diff
@@ -1,76 +1,65 @@
...
-  styleUrls: ['./feed-item-details.component.scss']
+  selector: "app-feed-item-details",
+  templateUrl: "./feed-item-details.component.html",
+  styleUrls: ["./feed-item-details.component.scss"],
 })
 export class FeedItemDetailsComponent implements OnInit {
+  @Input() feedItem: IFeedItem;
+  discussionChat: IDiscussion;
+
+  currImage = null;
```
Comment (akshyta) [inline projects/commudle-admin/src/app/feature-modules/external-feed/components/feed-item-details/feed-item-details.component.ts]: Are we using this variable anymore?

File: projects/commudle-admin/src/app/feature-modules/external-feed/components/external-feed/external-feed.component.html [hunk 2]
```diff
@@ -1,73 +1,67 @@
...
             Latest
           </button>
-          <button
-          (click)="getPopularPosts()">
+          <button (click)="getPopularPosts()">
             <nb-icon icon="flash" status="warning"></nb-icon>
             Hot
           </button>
-          <hr>
+          <hr />
```
Comment (akshyta) [inline projects/commudle-admin/src/app/feature-modules/external-feed/components/external-feed/external-feed.component.html]: Maybe we can remove this? What do you think?

File: projects/commudle-admin/src/app/feature-modules/external-feed/components/external-feed/external-feed.component.scss [hunk 3]
```diff
@@ -61,30 +60,21 @@
       border: $border-1;
       border-radius: $border-10;
       box-shadow: none;
-    }
-
-    .sorting{
-      // nb-icon {
-      //   font-size: $fs-40;
-      //   font-weight: $fw-500;
-      // }
```
Comment (akshyta) [inline projects/commudle-admin/src/app/feature-modules/external-feed/components/external-feed/external-feed.component.scss]: We need this class for highlighting the button selected, let's not remove it from the css.

**Review by akshyta — COMMENTED** [review]
Just a few clarifications, rest LGTM.

File: projects/commudle-admin/src/app/feature-modules/external-feed/components/feed-item-details/feed-item-details.component.ts (hunk 1 above)
Comment (anushika99) [reply projects/commudle-admin/src/app/feature-modules/external-feed/components/feed-item-details/feed-item-details.component.ts]: Removed

File: projects/commudle-admin/src/app/feature-modules/external-feed/components/external-feed/external-feed.component.html (hunk 2 above)
Comment (anushika99) [reply projects/commudle-admin/src/app/feature-modules/external-feed/components/external-feed/external-feed.component.html]: Removed

File: projects/commudle-admin/src/app/feature-modules/external-feed/components/external-feed/external-feed.component.scss (hunk 3 above)
Comment (anushika99) [reply projects/commudle-admin/src/app/feature-modules/external-feed/components/external-feed/external-feed.component.scss]: This was duplicate code

**Review by akshyta — APPROVED** [review]

**Pull request merged**